polkadot/runtime/rococo/Cargo.toml

```diff
@@ -323,6 +330,8 @@ try-runtime = [
 	"sp-runtime/try-runtime",
 ]
 
+disable-runtime-api = []
```

Comment:
Some comment would be nice.

Reply:
That makes sense and I will add descriptive comments.
One thing I'd like to confirm though, is this still required by Cumulus to access certain runtime(s)?

Reply:
Added some comments, let me know if they're descriptive enough.